components/SlateRenderer/components/Placeholder.tsx

```diff
@@ -0,0 +1,25 @@
+import type { PlaceholderNode } from '@prezly/slate-types';
+import { useCurrentStory } from '@prezly/theme-kit-nextjs';
+
+import { StoryPublicationDate } from '@/components';
+
+interface Props {
+    node: PlaceholderNode;
+}
+
+function Placeholder({ node }: Props) {
+    const currentStory = useCurrentStory();
+
+    if (!currentStory) {
```

Comment:
Just wondering, why `currentStory` could be null?

Reply:
`currentStory` is defined on `NewsroomContext` but is not always provided except for `/[slug]` and/or the preview `/s/[uuid]` pages.